packages/aura/src/components/menu-bar.css

```
@@ -0,0 +1,5 @@
+@container not style(--vaadin-menu-bar-gap) {
+  vaadin-menu-bar-button:not(:first-of-type) {
```

Consider using `[first-visible]` instead as we have `reverseCollapse` mode:

```suggestion
  vaadin-menu-bar-button:not([first-visible]) {
```